services/ui-src/src/components/forms/Form.tsx

```diff
@@ -15,9 +14,16 @@ import {
 } from "utils";
 import { AnyObject, FormJson, FormField, UserRoles } from "types";
 
-export const Form = ({ id, formJson, onSubmit, children, ...props }: Props) => {
+export const Form = ({
+  id,
+  formJson,
+  onSubmit,
+  formData = {},
```

Comment:
I think you should be able to leave this undefined since it's optional. Haven't tested it though so I might be missing something big-picture.